com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/common/ModelController.java

```diff
@@ -110,13 +113,17 @@ public void getOne(Context ctx, String modeluri) {
 	}
 
 	public void update(Context ctx, String modeluri) {
-		readEObject(ctx).ifPresentOrElse(
+		Optional<Resource> resource = readResource(ctx, "temp$update.json");
```

Comment:
I guess we should either rename the method or re-structure the code a bit as we are reading a resource here which potentially is created on the fly. We should tackle this track that via a separate issue in order to quickly get this PR in.